src/components/base/ExternalLink.vue

```diff
@@ -0,0 +1,17 @@
+<template>
```

Comment:
OOh thanks for adding in the external link component! What do you think of it?

Reply:
I like it! And it makes it more flexible to use social media icons if we get to the point that we need to link those kinds of things.